tests/forms_tests/tests/test_forms.py

```diff
@@ -3014,6 +3014,72 @@ class UserRegistration(Form):
             "</td></tr>",
         )
 
+    def test_widget_attrs_custom_describedby(self):
+        # Verify that aria-describedby provided to the widget overrides the default.
```

Comment:
To add more explanation, comments should state the expected behavior and omit prefixes like "Verify that" since all tests verify things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).